packages/storybook/stories/va-file-input-multiple-uswds.stories.jsx

```diff
@@ -38,10 +42,12 @@ const Template = ({
   errors,
   required,
   hint,
-  'enable-analytics': enableAnalytics,
+  enableAnalytics,
   vaMultipleChange,
   headerSize,
-  additional
```

Comment:
Renamed "additional" prop to "children" to be in line with va-file-input (single).